### DISCUSSION TITLE
Split Chat light mode setting

### DISCUSSION THREAD

**Discussion by JamieH93** [night/betterttv#6862] [Ideas]
Currently when you have split chat enabled, You can only pick for Dark.

However if you're not in theater mode/dark mode website wide, It'll make the chat look bad.
![image](https://github.com/night/betterttv/assets/8375853/6b2d6405-ec24-46b4-addd-1d13014f5e43)

I know most use dark mode system wide all the time however I feel it still needs a setting to fix this issue.
It could be a setting to auto disable in light mode or just another color picker.

**Comment by night** [comment]
Closing this one out — can't reproduce. The Split Chat color picker (Settings → Split Chat → Alternate Background Color) accepts any color, so you can pick a light shade if you're using Twitch's light theme. Please open a new discussion if you're still running into a case where it forces a dark-only color.
